### DISCUSSION TITLE
Installing cstools in our linux server, getting the following error

### DISCUSSION THREAD

**Discussion by vishnuvrv** [thoughtspot/cs_tools#138] [Q&A]
Hello we have a 8 node cluster for thoughtspot and when we tried installing cs_tools in on one of the nodes
curl \
    --silent --show-error --location-trusted \
    https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py \
    | python3 - --install

we are getting the following error:
[admin@myNode ~]$ curl --silent --show-error --location-trusted https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py  | python - --install
18:59:18 | Welcome to the CS Tools Bootstrapper!
         | Ideally, you will only need to run this one time, and then your environment can be fully managed by CS Tools itself.
         | If you run into any issues, please reach out to us on GitHub Discussions below.
         |
         |           GitHub: https://github.com/thoughtspot/cs_tools/issues/new/choose
         |
         | Missing '_updater.py', downloading from GitHub

Please help us resolve this

**Comment by boonhapus** [comment]
Hey @vishnuvrv it looks like you may not have outside Internet access from this cluster. Can you share the output of `python -m sysconfig` ? I'll build an offline binary for you.
